pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
```

Comment:
I see two options:
1. Remove the dynamic span name, and only create spans using `methodName`. No `fmt.Sprintf` or string concatenation, but it's a bit harder for humans to parse the spans (each span still gets a unique span ID, and the start time/duration are recorded, so still possible to piece together the sequencing).
2. Keep the dynamic name, replace the `fmt.Sprintf` with a string concatenation.

Which do you prefer?